### DISCUSSION TITLE
Enhancement: add symbolic link target in tooltip that shows when keeping cursor over the link

### DISCUSSION THREAD

**Discussion by m5k8** [doublecmd/doublecmd#1175] [Suggestions]
As in subject: add symbolic link target in tooltip that shows when keeping cursor over the link. Currently only modification time and size = 0 (useless item) is shown.

**Comment by alexx2000** [comment]
You can add it in the tooltips [options](https://doublecmd.github.io/doc/en/configuration.html#ConfigTooltips).

**Comment by m5k8** [comment]
Thanks for the suggestion, but as far as I can see, it doesn't allow to detect unix symbolic link.
I can configure tooltips for various types of files, but type of file is classified using some given file mask, for example by extension. Unix symbolic link is not detectable by file name, it's distinct file type on the system file level and can have absolutely any name.
Unless I'm still missing something?

**Comment by alexx2000** [comment]
You can do it via search template, press button with filter icon, choose "Define..." and create needed template.

**Comment by m5k8** [comment]
OK, thank you, I've found it and created required category mask, tooltip definition and applied expected category hint to it.
So it's universal, configurable and can serve any needs. Almost great, because frankly, there are too many hoops for getting such basic information about symbolic link.
I'd rather suggest to make it default behavior for symlink tooltips, otherwise it's a feature for very inquiring users only. I am, but most aren't.
